.changeset/thirty-grapes-change.md

```diff
@@ -0,0 +1,5 @@
+---
+"@digdir/designsystemet-react": patch
+---
+
+`fieldObserver`: removed logic that added aria-disabled to elements with `data-field` attribute if input was disabled
```

Comment:
```suggestion
**Field**: Removed logic that added `aria-disabled` to elements with `data-field` attribute if input was disabled
```
Since `fieldObserver` is not a consumer facing component, I think we should just put it down as `Field`